src/NMEA/Info.cpp

```diff
@@ -38,6 +38,9 @@ GPSState::Reset()
   satellites_used_available.Clear();
   satellite_ids_available.Clear();
   replay = false;
+  pdop = 0;
```

Comment:
What does zero mean here? Is this some kind of "magic" value? Why isn't this documented?
(Don't be distracted from the fact that `hdop` isn't documented either - that's old and bad code. I just don't accept adding more equally bad code.)

Reply:
Mmmh. It should be 1.0 as intial value. 0 is a bad initialization for that.
1.0 means that there is no dilution of precision, and is a much better default than 0 (no error at all, which is unrealistic)

HDOP, PDOP, and VDOP are well-defined terms in the NMEA specification. The only use of HDOP is in Logger/LoggerEPE where the meaning is epically described.
Here mentioning that the xDOP values are used for pass-through, and HDOP for error estimation should be sufficient.

Reply:
I agree, a value of 0 doesn't make sense and should not be used for initialization. 1.0 is not a good value either in my opinion, since that is the ideal state of this value (Ref.: https://en.wikipedia.org/wiki/Dilution_of_precision_(navigation)#Meaning_of_DOP_Values).

I would set it to 20 (or even better: 99). If you forward that value to other devices, they should discard it right away.

Reply:
Better: -1 is an impossible value, and an indictor that no value is present. When sending it out leave the field empty.

Reply:
@hor63 you mean like 2459e1e3378b7ca0a43e33ca0ce1ac759b97ae2f which I wrote 2 hours ago?

Reply:
:D Exactly. @Scumi should build on that.

Reply:
Thanks for updating that code, Max. Updated mine to align with the new support code.